src/features/scheduledEventHandler.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable @typescript-eslint/object-curly-spacing */
+/* eslint-disable @typescript-eslint/naming-convention */
+
+type Event = {
+  cron: string;
+  type: string;
+  scheduledTime: number;
+};
+type Env = {
+  [key: string]: string;
+  KV: KVNamespace;
+};
```

Comment:
We have duplicate `Env`, maybe move it to a `infras/worker.d.ts` or somewhere to put them?